app/main/posts/views/post-filters.service.js

```diff
@@ -75,7 +75,13 @@ function PostFiltersService(_, FormEndpoint, TagEndpoint, $q) {
     }
 
     function clearFilter(filterKey, value) {
-        filterState[filterKey] = getDefaults()[filterKey];
+        // if filter is in an array, we only want to remove that specific value
```

Comment:
So this just reverts to the previous behaviour?
Are there tests to avoid regressions in future?

Reply:
Yes, I am not sure why I removed this before since I was not using that function, must have been temporary(?) confusion. Tests added now...